helm/cluster-azure/values.yaml

```diff
@@ -18,6 +18,12 @@ network:
   serviceCIDR: 172.31.0.0/16
   podCIDR: "192.168.0.0/16"
   hostCIDR: "10.0.0.0/8"
+  mode: public
+
+bastion:
+  enabled: true
+  instanceType: Standard_D2s_v5
+  replicas: 1
```

Comment:
```suggestion
```

i would propose to have a fixed number of `1` and not make it configurable through the helm chart

Reply:
:+1: will do that